tidb-cloud/select-cluster-tier.md

```diff
@@ -10,30 +10,20 @@ The cluster tier determines the throughput and performance of your cluster.
 
 TiDB Cloud provides the following two options of cluster tiers. Before creating a cluster, you need to consider which option suits your need better.
 
-- [Developer Tier](#developer-tier)
+- [Serverless Tier](#serverless-tier)
 - [Dedicated Tier](#dedicated-tier)
 
-## Developer Tier
+## Serverless Tier
 
-The TiDB Cloud Developer Tier is a one-year free trial of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/), the fully managed service of TiDB. You can use Developer Tier clusters for non-production workloads such as prototype applications, hackathons, academic courses, or to provide a temporary data service for your datasets.
+The TiDB Cloud Serverless Tier is fully managed service of TiDB. It's still in beta phase and can not be used in production. However, you can use Serverless Tier clusters for non-production workloads such as prototype applications, hackathons, academic courses, or to provide a temporary data service for your datasets.
 
-Each Developer Tier cluster is a full-featured TiDB cluster and comes with the following:
-
-- 1 TiDB shared node
-- 1 TiKV shared node (with 1 GiB of OLTP storage)
-- 1 TiFlash shared node (with 1 GiB of OLAP storage)
-
-Developer Tier clusters run on shared nodes. Although each node is run in its own container on a virtual machine (VM), that VM is also running other TiDB, TiKV, or TiFlash nodes. As a result, shared nodes will have reduced performance when compared to standard, dedicated TiDB Cloud nodes. However, as all nodes are running in separate containers and have dedicated cloud disks, data stored in a Developer Tier cluster is isolated and will never be exposed to other TiDB clusters.
-
-For each TiDB Cloud account, you can use one complimentary Developer Tier cluster to use for one year. Although you can only run one Developer Tier cluster at a time, you can delete and recreate the cluster as many times as you wish.
-
-The one-year free trial begins the day the first Developer Tier cluster is created.
+For each TiDB Cloud account, you can use one complimentary Serverless Tier cluster to use during beta phase. Although you can only run one Serverless Tier cluster at a time, you can delete and recreate the cluster as many times as you wish.
```

Comment:
Serverless Tier's billing model hasn't been decided. But it would be very close to something like having some free quota for everyone and pay for extra usages.

Reply:
But what we are sure is that as long as Serverless Tier is in beta phase, we do not charge any money to our users.